modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/background/task/LayoutSetPrototypeImportBackgroundTaskExecutor.java

```diff
@@ -230,15 +257,26 @@ protected boolean isCancelPropagationImportTask() {
 	private static final Log _log = LogFactoryUtil.getLog(
 		LayoutSetPrototypeImportBackgroundTaskExecutor.class);
 
-	private int _importTaskIsolationLevel =
-		BackgroundTaskConstants.ISOLATION_LEVEL_GROUP;
+	@Reference
+	private BackgroundTaskManager _backgroundTaskManager;
+
+	@Reference
+	private ExportImportLocalService _exportImportLocalService;
+
+	@Reference
+	private LayoutSetLocalService _layoutSetLocalService;
+
+	@Reference
+	private LayoutSetPrototypeLocalService _layoutSetPrototypeLocalService;
 
 	private static class LayoutImportCallable implements Callable<Void> {
 
 		public LayoutImportCallable(
-			ExportImportConfiguration exportImportConfiguration, File file) {
+			ExportImportConfiguration exportImportConfiguration,
+			ExportImportLocalService exportImportLocalService, File file) {
 
 			_exportImportConfiguration = exportImportConfiguration;
+			_exportImportLocalService = exportImportLocalService;
 			_file = file;
```

Comment:
@vendeltoreki, please remove **static** and directly use _exportImportLocalService of LayoutSetPrototypeImportBackgroundTaskExecutor. After that, we may not pass ExportImportLocalService as param.